File: app/src/main/java/com/soumya/wwdablu/zomatobuddy/model/restaurant/R.java
package com.soumya.wwdablu.zomatobuddy.model.restaurant;

import com.google.gson.annotations.SerializedName;


public class R{

	@SerializedName("res_id")
	private int resId;

	public void setResId(int resId){
		this.resId = resId;
	}

	public int getResId(){
		return resId;
	}

	@Override
 	public String toString(){
		return 
			"R{" + 
			"res_id = '" + resId + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/soumya/wwdablu/hungry/network/model/search/R.kt
package com.soumya.wwdablu.hungry.network.model.search

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class R(
        @SerializedName("res_id")
        val resId: Int,

        @SerializedName("is_grocery_store")
        val isGroceryStore: Boolean,

        @SerializedName("has_menu_status")
        val hasMenuStatus: HasMenuStatus

) : Parcelable
